scheduler/scheduler.go

```diff
@@ -413,10 +415,10 @@ func (r *Registry) newFifoQueue(name string, c Config) *FifoQueue {
 	q := &FifoQueue{
 		config: c,
 		queue: &fifoQueue{
-			counter:        atomic.NewUint64(0),
+			counter:        &atomic.Int64{},
```

Comment:
Lets use `new(atomic.Int64)` for consistency

Reply:
Fixed in the next patch.